src/pages/LogsPage.tsx

```diff
@@ -87,65 +88,72 @@ export function LogsPage() {
   );
 
   return (
-    <SearchLayout
-      onRefresh={() => refetch()}
-      loading={isLoading || isFetching || isRefetching}
-      title={
-        <h1 className="text-xl font-semibold">
-          Logs{topology?.name ? `/${topology.name}` : ""}
-        </h1>
-      }
-      contentClass={`h-full p-6`}
-      extra={
-        <DropdownStandaloneWrapper
-          dropdownElem={<TimeRange name="time-range" />}
-          defaultValue={searchParams.get("start") ?? timeRanges[0].value}
-          paramKey="start"
-          className="w-44 mr-2"
-        />
-      }
-    >
-      <div className="flex flex-col space-y-6 h-full">
-        <div className="flex flex-row items-center w-full">
-          <FilterLogsByComponent />
+    <>
+      <Head prefix={topology?.name ? `Logs - ${topology.name}` : "Logs List"} />
```

Comment:
```suggestion
      <Head prefix={topology?.name ? `Logs - ${topology.name}` : "Logs"} />
```

We shouldn't have `List` in any of the titles